src/Widgets/Common/BackgroundWidget.php

```diff
@@ -87,7 +87,13 @@ private function createImageSettings($settings)
 
         $settings->createFile('customImagePath')
             ->withCondition('sourceType === "custom-image"')
-            ->withName('Widget.backgroundImageSource');
+            ->withName('Widget.backgroundImageSourceLabel')
+            ->withTooltip('Widget.backgroundImageSourceTooltip');
+
+        $settings->createFile('fallbackImagePath')
+            ->withCondition('sourceType === "custom-image"')
```

Comment:
```suggestion
            ->withCondition('sourceType === "custom-image" && /.?(\.webp)(?:$|\?)/.test(customImagePath)')
```

=> Auf diese Weise wird das Feld nur angezeigt, wenn für das erste Bild auch wirklich ein webp-Bild ausgewählt wurde